controllers/clustergroupupgrade_controller.go

```diff
@@ -406,6 +416,20 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 					)
 					nextReconcile = requeueImmediately()
 				} else {
+
+					// Check that we still have clusters to remediate since errors may have occured above
+					if len(clusters) == 0 {
```

Comment:
We don't need this progressing condition message, if backup failed for all the cluster as an example, backupCondition.Status will be metav1.ConditionFalse, preventing us from coming to this block

Reply:
Actually yeah we can only get here if the backup was at least partially successful, so I don't need this check at all